pkg/apis/apps/v1alpha1/subscriptionstatus_types.go

```diff
@@ -19,100 +19,133 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// SubscriptionStatus provides detailed status for all the resources that are deployed by the application in a cluster.
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope="Namespaced"
 // +kubebuilder:resource:shortName=appsubstatus
-// SubscriptionStatus defines the status of package deployments
 type SubscriptionStatus struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// Statuses represents all the resources deployed by the subscription per cluster
+	// Statuses represents all the resources deployed by the subscription in a cluster.
 	Statuses SubscriptionClusterStatusMap `json:"statuses,omitempty"`
 }
 
+// SubscriptionStatusList provides a list of SubscriptionStatus.
 // +kubebuilder:object:root=true
-// SubscriptionStatusList contains a list of SubscriptionStatus.
 type SubscriptionStatusList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []SubscriptionStatus `json:"items"`
 }
 
-// SubscriptionClusterStatusMap defines the status of packages in a cluster.
+// SubscriptionClusterStatusMap contains the status of deployment packages in a cluster.
 type SubscriptionClusterStatusMap struct {
```

Comment:
There are two places that describe the same SubscriptionClusterStatusMap structure. One is here. The other one is in Line 31.  It turns out only the description in line 31 is used in CRD. It seems more clear to add one description just above the the struct or the non-struct field.

So in this case, delete the description from line 31, only keep the description in line 43